blogposts/2020/3.18/announcing-sourcegraph-3.18.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Sourcegraph 3.18: Search across multiple revisions and non-master branches, custom search pages, campaigns now support GitLab, new C++ precise code intelligence"
+author: Laureen Hudson
+publishDate: 2020-MM-20T10:00-07:00
+tags: [blog]
+slug: sourcegraph-3.18
+heroImage: /blog/3.18-release-blog-img.jpg
+published: true
+---
+
+We’re improving Sourcegraph universal code search continuously, to help you solve the Big Code problems you face, all day every day. Here's how we're doing it.
+
+-   [Indexed non-master branches](https://about.sourcegraph.com/blog/indexed-non-master-branches)
+
+    Often, you need to search code that isn’t in your default branch, like long-lived release branches or important tags. You can now index multiple branches (not just the default branch) to make these kinds of searches much faster.
+
+-   [Search across multiple revisions](https://about.sourcegraph.com/blog/search-multiple-revisions)
+
+    You can now search across multiple revisions of the same repository by listing multiple branch names (or other revision specifications) separated by `:` in your query. This helps you understand the differences between code at different branches (especially for release branches that have diverged).
+
+-   [Small but mighty new features](https://about.sourcegraph.com/blog/small-and-mighty-features)
+
+    Based on lots of feedback, we added support for campaigns in GitLab. We’ve also improved privacy feedback when adding repositories, and created custom search pages for Sourcegraph Cloud.
+
+-   [New C++ precise code intelligence solution](https://about.sourcegraph.com/blog/c-plus-plus-code-intel)
+
+    We wrote a new LSIF indexer based on clangd. This means that developers using C and C++ now have access to precise code intelligence. Developers using the original C++ LSIF indexer based on DXP will see noticeable improvements with this new implementation; faster than before, and now with hovers.
```

Comment:
The `; faster than before, and now with hovers.` part of the sentence sounds weird to me. It is a sentence fragment. It would sound better if it was changed to something like `: it's faster and has hovers.`

Reply:
I liked preserving Garo's style there, but I'm happy to make the change.

Reply:
I liked Garo's voice here, but have changed.